packages/feedback-service/src/resolver.ts

```diff
@@ -11,7 +11,6 @@ export const FeedbackResolver = {
       return Feedback.find()
         .sort({ 'timestamp': -1 })
         .then(response => {
-          let newresponse: any[] = [];
           const createdBy = response.map(res => `rhatUUID_${res.createdBy}:getUsersBy(rhatUUID:"${res.createdBy}") { _id name title uid rhatUUID isActive}`);
```

Comment:
This might give an error for duplicate aliases.
Have you tested it for cases where the same user has submitted multiple feedbacks?

Check if it throws any error for multiple feedbacks with same `createdBy` or `updatedBy`.

Reply:
If it does throw an error, you may have to get the unique rhatUUIDs first, and then pass them on to the GraphQL request.

Reply:
Yes i tested it and tested it again now duplicated aliases. does not through error.